pkg/controllers/node/termination/controller.go

```diff
@@ -83,7 +83,7 @@ func (c *Controller) Finalize(ctx context.Context, node *v1.Node) (reconcile.Res
 		return reconcile.Result{}, fmt.Errorf("deleting nodeclaims, %w", err)
 	}
 	if err := c.terminator.Cordon(ctx, node); err != nil {
```

Comment:
I forget what the word "cordon" means here but is it still appropriate to call this cordoning if it's not using the standard upstream K8s taint?

Reply:
I went back and forth on this one, and wanted to keep it the same as is, since we're still cordoning nodes. But I also changed the log, so I'm just going to change this to `Taint` now since Unschedulable is still a taint.